resources/fixtures/config/analytics.yml

```diff
@@ -0,0 +1,2 @@
+provider: google
```

Comment:
I think this should be `google-analytics`, since that's the actual name of the provider. If for some reason Google decide to release a secondary analytics platform, this would be confusing